diff --git a/contrib/Makefile.in b/contrib/Makefile.in
index 72e4d4a9..62415152 100644
--- a/contrib/Makefile.in
+++ b/contrib/Makefile.in
@@ -84,52 +84,3 @@ $(CGI_O): $(CGI_C)
 
 %.cgi : %.c
 	$(CC) $(CFLAGS) $(LDFLAGS) $< $(CGI_O) -o $@
-
-##############################################################################
-# making similiar EPEL rpm for Fedora/RHEL/CentOS 
-
-ifneq ("$(wildcard /etc/redhat-release)","")
-
-ARCH ?= $(shell uname -m)
-ifeq ($(ARCH),x86_64)
-RPM_ARCH := x86_64
-else
-    ifeq ($(ARCH),i686)
-RPM_ARCH := i386
-    else
-        ifeq ($(ARCH),aarch64)
-RPM_ARCH := aarch64
-        else
-$(error Unknown arch "$(ARCH)".)
-        endif
-    endif
-endif
-
-# Create our own tarball and put ino rpmbuild/SOURCES
-localsrc-rpm:
-	# create nagios tar ball.
-	@(cd ..;rm -f nagios-@VERSION@)
-	@(cd ..;ln -s . nagios-@VERSION@)
-	@(cd ..;tar zhcf nagios-@VERSION@.tar.gz --exclude nagios-@VERSION@/nagios-@VERSION@.tar.gz --exclude nagios-@VERSION@/nagios-@VERSION@ --exclude RCS --exclude CVS --exclude build-* --exclude *~ --exclude .git* nagios-@VERSION@/)
-	@(cd ..;rm -f nagios-@VERSION@)
-	# build the rpm using rpmbuild from ./rmbuild as topdir
-	@rm -rf rpmbuild && mkdir -p ./rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
-	@cp  ../nagios-@VERSION@.tar.gz rpmbuild/SOURCES/nagios-@VERSION@.tar.gz
-	@cp epel-patches/* ./rpmbuild/SOURCES
-	@rpmbuild -ba --define "_topdir ${PWD}/rpmbuild"   epel-nagios.spec
-	@cp  rpmbuild/RPMS/$(RPM_ARCH)/*.rpm .
-	@cp rpmbuild/SRPMS/*.src.rpm .
-	@ls -l *.rpm
-
-# tarball from https://github.com/NagiosEnterprises/nagioscore/releases/download/nagios-%{version}/nagios-%{version}.tar.gz
-rpm: init-epelrpm  build-epelrpm
-	@mv rpmbuild/RPMS/$(RPM_ARCH)/*.rpm .
-	@ls -l *.rpm
-init-epelrpm: 
-	# build the rpm using rpmbuild from ./rmbuild as topdir
-	@rm -rf ./rpmbuild && mkdir -p ./rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
-	@spectool -C ./rpmbuild/SOURCES -g epel-nagios.spec
-	@cp epel-patches/* ./rpmbuild/SOURCES
-build-epelrpm:
-	@rpmbuild -ba --define "_topdir ${PWD}/rpmbuild"  epel-nagios.spec
-endif
